updatecli/values.yaml

```diff
@@ -5,3 +5,18 @@ github:
   branch: "main"
   owner: "jenkins-infra"
   repository: "azure"
+
+# Also used by terraform in locals.tf
+end_dates:
+  infra_ci_jenkins_io_fileshare_serviceprincipal_writer:
+    end_date: "2024-06-20T23:00:00Z"
+    concern: "contributors.jenkins.io"
+  infraci_docs_jenkins_io_fileshare_serviceprincipal_writer:
+    end_date: "2024-08-07T23:00:00Z"
+    concern: "docs.jenkins.io"
+  infraci_pluginsjenkinsio_fileshare_serviceprincipal_writer:
+    end_date: "2024-07-27T00:00:00Z"
+    concern: "plugins.jenkins.io"
+  infraci_stats_jenkins_io_fileshare_serviceprincipal_writer:
+    end_date: "2023-09-19T23:00:00Z"
```

Comment:
Found the inversion

```suggestion
    end_date: "2024-09-19T23:00:00Z"
```

Reply:
Ref. https://github.com/jenkins-infra/azure/pull/741/files#diff-1a27c7a4b6293913f23d9922f0f57cb36a737ae6d88cdbbaa49a463b2a2cc360L7

Reply:
Damn, that was the change I made to test the manifest, commited by mistake 😅